custodian/policies/ec2/ec2-delete-marked.yml

```diff
@@ -5,18 +5,19 @@ policies:
       schedule: "rate(1 day)"
       role: arn:aws:iam::{account_id}:role/CustodianDeleteMarkedEC2
     resource: aws.ec2
-    comments: |
+    description: |
       Delete any EC2 instances scheduled for deletion
```

Comment:
Should the description be changed to: "Terminate any EC2 ...", to match the wording of the action descriptions?

Reply:
thanks, updated as suggested